### ISSUE TITLE
unable to generate key

### ISSUE THREAD

**Issue by ihubgit** [DimensionDev/Tessercube-Meta#13]
hi I'm using version 1.0 (5) and am stuck in the stage of creating a key pair. Running iOS 12.2. After entering wished username and password I'm getting the following error :
Password should be minimum 8 characters at least 1 Alphabet and 1 number
Now my password respects these conditions and yet no way to move onward ...

**Comment by ihubgit** [comment]
ok just tried with the same version number and with indications for pw on reddit, using similar layout. This time it works. What might be the problem then, is that i include the "@" sign in my pw. Does this mean only letters and numerals are accepted?

**Comment by Tedko** [comment]
Yeah I suppose we should support strong Pw definitely. Will update this soon.

**Comment by ihubgit** [comment]
Good.
Also posted follow-up on reddit.

**Comment by neruthes** [comment]
Hi ihubgit, thank you for the information you provided to help us investigate the bug. And due to security considerations, I have deleted one of your comment to make sure that your password preference is no longer visible to anyone else. This comment will be deleted soon so that no one else would passively figure out that you did leak your password preference in a deleted comment.

**Comment by neruthes** [comment]
@Tedko This is a bug; please do not rendering it a feature request.
